go.mod

```diff
@@ -93,7 +93,7 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.38.0
 // currently fails because Thanos isn't merging release branches to main branch, and Go modules system is then
 // confused about which version is the latest one. v0.22.0 was released in July, but latest tag reachable from main
 // is v0.19.1. We pin version from early December here. Feel free to remove when updating to later version.
-replace github.com/thanos-io/thanos v0.22.0 => github.com/thanos-io/thanos v0.19.1-0.20211208205607-d1acaea2a11a
+replace github.com/thanos-io/thanos v0.22.0 => github.com/someshkoli/thanos v0.13.1-0.20220130010113-1056df839e44
```

Comment:
is this a left over?, we can't use your fork for this.

Reply:
Both the or are dependent on each other 🤔🤔 not sure how to handle this properly for ci to pass